micrometer-core/src/main/java/io/micrometer/core/instrument/datadog/DatadogRegistry.java

```diff
@@ -38,7 +38,7 @@
  *
  * @author Jon Schneider
  */
-final class DatadogRegistry extends AbstractStepRegistry {
+public final class DatadogRegistry extends AbstractStepRegistry {
```

Comment:
The original intent here was to hide Spectator as an implementation detail. Might it be used directly?